lib/adapters/REST/endpoints/organization.ts

```diff
@@ -1,18 +1,23 @@
 import type { AxiosInstance } from 'contentful-sdk-core'
-import { CollectionProp, GetOrganizationParams } from '../../../common-types'
+import { CollectionProp, GetOrganizationParams, PaginationQueryParams } from '../../../common-types'
 import { OrganizationProp } from '../../../entities/organization'
 import { RestEndpoint } from '../types'
 import * as raw from './raw'
 
-export const getMany: RestEndpoint<'Organization', 'getMany'> = (http: AxiosInstance) => {
-  return raw.get<CollectionProp<OrganizationProp>>(http, `/organizations`)
+export const getMany: RestEndpoint<'Organization', 'getMany'> = (
+  http: AxiosInstance,
+  params?: PaginationQueryParams
+) => {
+  return raw.get<CollectionProp<OrganizationProp>>(http, `/organizations`, {
+    params: params?.query,
+  })
 }
 
 export const get: RestEndpoint<'Organization', 'get'> = (
   http: AxiosInstance,
   params: GetOrganizationParams
 ) => {
-  return getMany(http).then((data) => {
+  return getMany(http, { query: { limit: 100 } }).then((data) => {
```

Comment:
Hmm... do we really need to pass this query param here given it's a `get` operation that should only return 1 item?

Reply:
I am not very sure but I thought that without it, setting an increased limit for `getAll()` could backfire later when `get()` is called because it would still get 25 orgs

Reply:
right but if you continue reading the code it's expecting to find a single object in `data.items` that matches the `params.organizationId`, meaning this is using the getMany to getOne or None

Reply:
Yeah, I understand, but my point is that without the change that I made, if someone has let's say 26 orgs and then they use get() in attempt to search for the org that is last (26th)? They will get an error, although the org would be there. With my custom limit it will fetch 100 orgs and there's a higher chance that none of the customers will have an account that would have more than 100 orgs.

Ideally we should have smth like fetchEntireDataset in there, but it comes from a different package, which I can't import cause it would be a cyclic ref.

Reply:
OH! Duh I see what you're saying now. We aren't passing in a filter by sys.id in this query

Reply:
(to which I would argue Someday® we should make sure the API supports it and then pass in a `sys.id[in]` filter here so we only have to fetch the one)

Reply:
if you don't mind, I would like to delegate it to the team owning the endpoint as it's a bit of a larger effort than what I am trying to achieve in this PR